frontend/src/components/public_group/AmountPicker.js

```diff
@@ -44,7 +44,7 @@ export default class AmountPicker extends React.Component {
         )}
 
         <div className='max-width-1 mx-auto' style={{position: 'absolute', left: '0', right: '0', margin: '-25px auto'}}>
-          {!hasStripe && (
+          {(!hasStripe || (hasPaypal && hasStripe)) && (
```

Comment:
That's right. That's why both `hasPaypal` and `hasStripe` should never be true at the same time *in our current setup*. We'll update the logic when we accept both for a collective.